src/platform/datapath_raw_xdp.c

```diff
@@ -111,6 +112,248 @@ CxPlatDataPathRecvDataToRecvPacket(
 CXPLAT_THREAD_CALLBACK(CxPlatXdpWorkerThread, Context);
 CXPLAT_THREAD_CALLBACK(CxPlatXdpExtraWorkerThread, Context);
 
+QUIC_STATUS
+CxPlatGetInterfaceRssQueueCount(
+    _In_ uint32_t InterfaceIndex,
+    _Out_ uint8_t* Count
+    )
+{
+    HRESULT hRes;
+    IWbemLocator *pLoc = NULL;
+    IEnumWbemClassObject *pEnum = NULL;
+    IWbemServices *pSvc = NULL;
+    DWORD ret = 0;
+    uint8_t cnt = 0;
+    NET_LUID if_luid = { 0 };
+    WCHAR if_alias[256 + 1] = { 0 };
+
+    ret = ConvertInterfaceIndexToLuid(InterfaceIndex, &if_luid);
+    if (ret != NO_ERROR) {
+        QuicTraceEvent(
+            LibraryErrorStatus,
+            "[ lib] ERROR, %u, %s.",
+            ret,
+            "ConvertInterfaceIndexToLuid");
+        return HRESULT_FROM_WIN32(ret);
+    }
+
+    ret = ConvertInterfaceLuidToAlias(&if_luid, if_alias, RTL_NUMBER_OF(if_alias));
+    if (ret != NO_ERROR) {
+        QuicTraceEvent(
+            LibraryErrorStatus,
+            "[ lib] ERROR, %u, %s.",
+            ret,
+            "ConvertInterfaceLuidToAlias");
+        return HRESULT_FROM_WIN32(ret);
+    }
+
+    // Step 1: --------------------------------------------------
+    // Initialize COM. ------------------------------------------
+    hRes =  CoInitializeEx(0, COINIT_MULTITHREADED);
+    if (FAILED(hRes)) {
+        QuicTraceEvent(
+            LibraryErrorStatus,
+            "[ lib] ERROR, %u, %s.",
+            hRes,
+            "CoInitializeEx");
+        return hRes;
+    }
+
+    // Step 3: ---------------------------------------------------
```

Comment:
Are we missing one step? Previous step is 2.

Reply:
I copied this code. 😄

Reply:
I see. A comment just for numbering seems unnecessary.

Reply:
I fixed the comment.

Reply:
BTW, this code is all temporary. It will be updated once XDP RSS API is available.